docs/content/manual/release-specific/v1.0.2/upgrade-conflict-handling.md

```diff
@@ -0,0 +1,12 @@
+### New installation:
+1. create a large cluster of many nodes (about 30 nodes)
+1. Install Longhorn to v1.0.1
+
+### Upgrading from old version:
+1. create a large cluster of many nodes (about 30 nodes)
+1. Install Longhorn v1.0.0
+1. Upgrade Longhorn to v1.0.1
+
+**Success if:** install/upgrade successfully after maximum 15 minutes. 
+
+**Fail if:** The upgrading is blocked after about 17-20 managers are up.
```

Comment:
The fix is not in v1.0.1. The validation steps above are not accurate.

Also, we need to mention this is a race condition so we need to try multiple times to validate the fix.

Reply:
Sorry, my bad. Fixed

Reply:
We can move this to the release testing. This is important to make sure there is no issue with Longhorn installation on larger cluster.

Reply:
Yes, moved it to `pre-release` tests